README.md

```diff
@@ -28,12 +28,28 @@ Die [Koalitionsvereinbarung für die Stadtratsperiode 2020 –2026 zwischen Ober
 Dieses Konzept definiert [Free Open Source Software](https://de.wikipedia.org/wiki/Free/Libre_Open_Source_Software) (FOSS) für die Landeshauptstadt München auf Basis des Koalitionsvertrags.
 Es definiert Voraussetzungen, Abhängigkeiten, Ausschlüsse und Zielrichtung von Entwicklung, Beschaffung und Betrieb von OpenSource Software.
 
+## Offene Standards als Basis
 
-## Offene Standards
+## Verwendung von FOSS in der LHM
 
-## Keine Herstellerabhängigkeiten
+### Client-Betriebssysteme und Office-Anwendungen
 
-## Client-Betriebssysteme und Office-Anwendungen
+### Kommunale Software
+
```

Comment:
Finde die Aufteilung in Unterkapitel wie gesagt sehr gut. Aber was ist mit dem Unterkapitel "Kommunale Software" gemeint, was soll da rein?

Reply:
Ich denke man muss hier zwischen "normaler Software" (die im Prinzip jeder nutzen kann, und kommunaler Software trennen. Komunale Softwre ist so etwas wie ein Einwohnermeldewesen-, KFZ-, Waffenverwaltung-, Führerschein-,  oder Ausländerwesen-Programm. Das besondere an diesem Markt ist, dass er von einigen wenigen großen Anbietern dominiert wird (wenn man man von so Nischen wie Friedhofsverwaltung, usw. absieht) und Open Source keine, bzw. eine sehr untergeordnete Rolle spielt. Im besten Fall gibt es Anbieter, die halbwegs moderne Systeme haben, in denen dann auch OSS Komponenten verbaut sind. Ein großer Teil hat einen Fokus auf Windows Fat Client Programme, die vor 15 - 20 Jahren als halbwegs modern gelten konnten.

Reply:
Ok, das leuchtet mir ein. Vielleicht magst Du das gleich so rein schreiben?

Reply:
Wir haben jetzt ein Unterkapitel "Eigenentwicklungen" und ein weiteres für "Beschaffungen", ist das damit abgedeckt?